dev/com.ibm.ws.concurrent.persistent_fat_multiple/publish/servers/com.ibm.ws.concurrent.persistent.fat.multiple/server.xml

```diff
@@ -20,24 +20,42 @@
   <include location="../fatTestPorts.xml"/>
 
   <persistentExecutor id="executor1" jndiName="concurrent/executor1" pollInterval="4s" retryLimit="1" pollSize="100"/>
-
   <persistentExecutor id="executor2" jndiName="concurrent/executor2" taskStoreRef="defaultDatabaseStore" pollInterval="4s" retryLimit="1"/>
 
   <databaseStore id="defaultDatabaseStore" keyGenerationStrategy="SEQUENCE"/>
+  
+  <library id="FATJDBCLib">
+    <fileset dir="${shared.resource.dir}/derby" includes="derby.jar"/>
+  </library>
+  
+  <library id="AnonymousJDBCLib">
+    <fileset dir="${shared.resource.dir}/jdbc" includes="driver.jar"/>
+  </library>
 
   <!-- database for scheduled tasks, also accessible to the application -->
-  <dataSource id="DefaultDataSource" fat.modify="true" containerAuthDataRef="authx">
+  <dataSource id="DefaultDataSource" jndiName="jdbc/DefaultDataSource" containerAuthDataRef="authx">
     <jdbcDriver libraryRef="FATJDBCLib"/>
     <properties.derby.embedded createDatabase="create" databaseName="memory:persistmultidb"/>
   </dataSource>
-
-  <library id="FATJDBCLib">
-    <fileset dir="${shared.resource.dir}/derby" includes="derby.jar"/>
-  </library>
+  
+    <!-- anonymous database for scheduled tasks, also accessible to the application -->
```

Comment:
why is AnonymousDataSource and AnonymousDataSource_DB2 needed? We should be able to get by with just DefaultDataSource for everything